CHANGELOG.md

```
@@ -3,6 +3,40 @@
+### Changes
+
+---
+
+Packages with breaking changes:
+
+ - There are no breaking changes in this release.
+
+Packages with other changes:
+
+ - [`stripe_android` - `v14.0.1`](#stripe_android---v1401)
+ - [`flutter_stripe` - `v14.0.1`](#flutter_stripe---v1401)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)